SmartContracts.NET/StrongForce.Core/Contract.cs

```diff
@@ -66,7 +66,7 @@ protected bool CheckPermission(ActionContext context, Action action)
 				checkedAction = forwardAction.FinalAction;
 			}
 
-			var permission = new Permission(checkedAction.GetType());
+			var permission = checkedAction.GetType();
```

Comment:
Use explicit types always, when the type is not explicitly written on the same line. There are other places where we're not doing this, and I think we should fix that.

Reply:
I think it would be better if all instances of this problem are resolved in a separate PR.